### DISCUSSION TITLE
(re)Downloading all likes

### DISCUSSION THREAD

**Discussion by fallingman** [TumblThreeApp/TumblThree#562] [General]
Hi firstly want to say TumblThree is a great app so thank you. 

Secondly I'm trying to redownload all my likes on a new laptop, but it seems to only download new ones, which makes me think something somewhere is keeping track of what's been downloaded before: please can anyone advise how I reset and get everything? MTIA - fallingman

**Comment by thomas694** [comment]
Hello,
the index files contain entries for items that have already been downloaded, so you must delete the `Likes_files.tlb` file (with the app closed) to be able to download them again.

**Reply by fallingman** [reply]
Thanks for the reply - but deleting that file causes the overall Likes blog to fail loading and not appear. I tried editing the data in the Likes_files.tlb file to remove the list of entries but that fails to load the blog too.

**Reply by thomas694** [reply]
Oops, deleting the whole file no longer works. But you can remove the `Entries` from the file, either completely or emptying the list only (`"Entries": [ ],`). I just tried both.
If you can't get it working, you can remember your blog settings, delete both files and add your Likes blog again. You also need to empty the `Download location` field after re-adding the blog as it finds your existing folder and starts with a new one (e.g. `Likes_2`)
